From 9516859b9fdb2899a37116b55200fb04f7726b45 Mon Sep 17 00:00:00 2001
From: Daniel Stone <daniels@collabora.com>
Date: Tue, 8 Feb 2022 22:39:42 +0000
Subject: [PATCH] tests: Add dependency on screenshooter client protocol

Given that the test-helper code relies on the screenshooter protocol,
make sure it's available for us to build, and the dependency ensures we
build in order.

Fixes: #588

Signed-off-by: Daniel Stone <daniels@collabora.com>
---
 tests/meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index d8e96e77d..955c4226c 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -29,8 +29,9 @@ lib_test_client = static_library(
 		'weston-test-client-helper.c',
 		'weston-test-fixture-compositor.c',
 		weston_test_client_protocol_h,
-		weston_screenshooter_protocol_c,
 		weston_test_protocol_c,
+		weston_screenshooter_client_protocol_h,
+		weston_screenshooter_protocol_c,
 		viewporter_client_protocol_h,
 		viewporter_protocol_c,
 		'color_util.h',
-- 
GitLab

